package.json

```diff
@@ -28,13 +28,9 @@
     "husky": "^8.0.0",
     "lint-staged": "^13.1.0"
   },
-  "lint-staged": {
-    "**/*.{js,jsx}": [
-      "npm run lint:fix:api",
-      "npm run lint:fix:client"
-    ]
-  },
+  "lint-staged": {},
   "dependencies": {
-    "firebase": "^9.17.0"
+    "firebase": "^9.17.0",
+    "react-icons": "^5.0.1"
```

Comment:
You have already added the _react-icons_ library in the `frontend/package.json` file.